types/webextension-polyfill/namespaces/storage.d.ts

```diff
@@ -186,15 +186,15 @@ export namespace Storage {
     /**
      * Object mapping each key that changed to its corresponding $(ref:storage.StorageChange) for that item.
      */
-    interface StorageAreaOnChangedChangesType extends StorageChange {
-        [s: string]: unknown;
+    interface StorageAreaOnChangedChangesType {
```

Comment:
Why are these not generics with defaults passing the type parameters to StorageChange?